op-e2e/actions/proofs/isthmus_fork_test.go

```diff
@@ -133,143 +154,119 @@ func TestWithdrawlsRootPreCanyonAndIsthmus(gt *testing.T) {
+			dc.L2GenesisIsthmusTimeOffset = &two
+			dc.L1PragueTimeOffset = &zeroHex64
 		}
-		sequencer.ActL2EndBlock(t)
+		env := helpers.NewL2FaultProofEnv(t, testCfg, tp, helpers.NewBatcherCfg(), setIsthmusTime)
+		withdrawalTx, withdrawalTxBlock, totalBlocks := testCfg.Custom.withdrawalTx, testCfg.Custom.withdrawalTxBlock, testCfg.Custom.totalBlocks
+		log := testlog.Logger(t, log.LvlDebug)
+		require.NoError(t, env.Dp.DeployConfig.Check(log), "must have valid config")
+
+		sequencer, engine := env.Sequencer, env.Engine
+
+		// start op-nodes
+		sequencer.ActL2PipelineFull(t)
+
+		verifyPreIsthmusHeaderWithdrawalsRoot(gt, engine.L2Chain().CurrentBlock())
+
+		ethCl := engine.EthClient()
+		for i := 1; i <= totalBlocks; i++ {
+			var tx *types.Transaction
+
+			sequencer.ActL2StartBlock(t)
+
+			if withdrawalTx && withdrawalTxBlock == i {
+				l2withdrawer, err := bindings.NewL2ToL1MessagePasser(predeploys.L2ToL1MessagePasserAddr, ethCl)
+				require.NoError(t, err, "binding withdrawer on L2")
+
+				// Initiate Withdrawal
+				// Bind L2 Withdrawer Contract and invoke the Receive function
+				l2opts, err := bind.NewKeyedTransactorWithChainID(env.Alice.L2.Secret(), new(big.Int).SetUint64(env.Dp.DeployConfig.L2ChainID))
+				require.NoError(t, err)
+				l2opts.Value = big.NewInt(500)
+				tx, err = l2withdrawer.Receive(l2opts)
+				require.NoError(t, err)
+
+				// force-include the transaction, also in upgrade blocks
+				engine.ActL2IncludeTxIgnoreForcedEmpty(env.Alice.Address())(t)
+			}
+			sequencer.ActL2EndBlock(t)
+
+			if withdrawalTx && withdrawalTxBlock == i {
+				// wait for withdrawal to be included in a block
+				receipt, err := geth.WaitForTransaction(tx.Hash(), ethCl, 10*time.Duration(env.Dp.DeployConfig.L2BlockTime)*time.Second)
+				require.NoError(t, err, "withdrawal initiated on L2 sequencer")
+				require.Equal(t, types.ReceiptStatusSuccessful, receipt.Status, "transaction had incorrect status")
+			}
+		}
+		rpcCl := engine.RPCClient()
 
-		if withdrawalTx && withdrawalTxBlock == i {
-			// wait for withdrawal to be included in a block
-			receipt, err := geth.WaitForTransaction(tx.Hash(), ethCl, 10*time.Duration(dp.DeployConfig.L2BlockTime)*time.Second)
-			require.NoError(t, err, "withdrawal initiated on L2 sequencer")
-			require.Equal(t, types.ReceiptStatusSuccessful, receipt.Status, "transaction had incorrect status")
+		// we set withdrawals root only at or after isthmus
+		if totalBlocks >= isthmusOffset {
+			verifyIsthmusHeaderWithdrawalsRoot(gt, rpcCl, engine.L2Chain().CurrentBlock(), true)
 		}
-	}
-	rpcCl := engine.RPCClient()
 
-	// we set withdrawals root only at or after isthmus
-	if totalBlocks >= isthmusOffset {
-		verifyIsthmusHeaderWithdrawalsRoot(gt, rpcCl, engine.L2Chain().CurrentBlock(), true)
+		l2Safe := sequencer.L2Safe()
+		env.RunFaultProofProgram(t, l2Safe.Number, testCfg.CheckResult, testCfg.InputParams...)
 	}
-}
 
-func TestWithdrawlsRootPostIsthmus(gt *testing.T) {
```

Comment:
I did not migrate this test, since it seems to me that it is covered by `TestWithdrawalsRootBeforeAtAndAfterIsthmus` (which is fully migrated to `Test_ProgramAction_WithdrawalsRootBeforeAtAndAfterIsthmus`).

Reply:
we should delete this test if it doesn't give any additional coverage